app/models/test_track/analytics_event.rb

```diff
@@ -10,9 +10,9 @@ def initialize(assignment)
 
     def name
       if assignment.feature_gate?
-        'FeatureGateExperienced'
+        'feature_gate_experienced'
       else
-        'SplitAssigned'
+        'split_assigned'
```

Comment:
do these mixpanel [specs](https://github.com/Betterment/test_track_rails_client/blob/master/spec/models/test_track/analytics/mixpanel_client_spec.rb#L12) need some updating too?

Also, just curious whether this "breaks" the continuity of some funnels being tracked in mixpanel?  Do folks need to update what they're tracking to look at either `SplitAssigned` or `split_assigned` when this lands in the apps?

Reply:
I can update these specs too though they're not actually hurting anything, just a little less realistic.